src/test/ajaxWrapper.test.js

```diff
@@ -39,6 +39,20 @@ describe("ajaxWrapper", function () {
       });
     });
 
+    it("defaults to GET when no method is supplied", function (done) {
+      ajaxWrapper({
+        url: config.apiURL
+      })
+      .success(function (response) {
+        expectAsync(function () {
+          expect(response.body.name).to.equal("Marathon");
+        }, done);
+      })
+      .error(function () {
```

Comment:
It's more like debugging-friendliness than anything. If I, for example, did a `this.server.setup(xY, 404)` in the same test and "forgot" to restore the desired 200 status then I would see an error in the tests. This also seems the convention in the file for all other tests, so I kept it. However I agree with you these lines may be adding unnecessary clutter.